{"name": "test_project_readiness", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <test_login.Test001Login object at 0x7fe3834f9820>\nsetup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"666365245261001d53d9c513a901a35e\")>\n\n    def test_project_readiness(self,setup):\n>       rad = Readiness(setup)\n\ntestCases/test_login.py:149: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npageObjects/ProjectDetailsForm/ProjectReadiness.py:140: in __init__\n    self.goto()\npageObjects/ProjectDetailsForm/ProjectReadiness.py:143: in goto\n    link = utils.explicit_wait_present(self.driver,self.project_readiliness_xpath)\nUtils/ComplexElement.py:14: in explicit_wait_present\n    element = wait.until(EC.presence_of_element_located((By.XPATH,locator)))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"666365245261001d53d9c513a901a35e\")>\nmethod = <selenium.webdriver.support.expected_conditions.presence_of_element_located object at 0x7fe3834f9250>, message = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n../../../.local/lib/python3.8/site-packages/selenium/webdriver/support/wait.py:80: TimeoutException"}, "start": 1629274733638, "stop": 1629274743722, "uuid": "0daba01d-69be-47e0-adf1-be8d0892e7ee", "historyId": "217a5efdff5c06fd97fa7582f1e89b36", "testCaseId": "abe187d16909ea58817f756255ef2127", "fullName": "testCases.test_login.Test001Login#test_project_readiness", "labels": [{"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test001Login"}, {"name": "host", "value": "bipin-Nitro-AN515-54"}, {"name": "thread", "value": "81834-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_login"}]}